src/modules/FioRequest/components/FioRequestRow.js

```diff
@@ -0,0 +1,160 @@
+// @flow
+
+import type { EdgeDenomination } from 'edge-core-js'
+import React, { Component } from 'react'
+import { Image, TouchableHighlight, View } from 'react-native'
+
+import fioRequestsIcon from '../../../assets/images/fio/fio_sent_request.png'
+import { intl } from '../../../locales/intl'
+import s from '../../../locales/strings'
+import { styles as requestListStyles } from '../../../styles/scenes/FioRequestListStyle'
+import styles from '../../../styles/scenes/TransactionListStyle'
+import THEME from '../../../theme/variables/airbitz'
+import type { FioRequest } from '../../../types/types'
+import T from '../../UI/components/FormattedText/index'
+import { isRejectedFioRequest, isSentFioRequest } from '../util'
+
+type Props = {
+  fioRequest: FioRequest,
+  fiatSymbol: string,
+  fiatAmount: (currencyCode: string, amount: string) => string,
+  onSelect: FioRequest => void,
+  displayDenominations: { [string]: EdgeDenomination },
+  isHeaderRow?: boolean,
+  isLastOfDate?: boolean,
+  isSent?: boolean
+}
+
+class FioRequestRow extends Component<Props> {
+  underlayColor = THEME.COLORS.ROW_PRESSED
+  minimumFontScale = 0.6
+
+  static defaultProps: Props = {
+    fioRequest: {
+      fio_request_id: '',
+      content: {
+        payee_public_address: '',
+        amount: '',
+        token_code: '',
+        chain_code: '',
+        memo: ''
+      },
+      payee_fio_address: '',
+      payer_fio_address: '',
+      status: '',
+      time_stamp: ''
+    },
+    fiatSymbol: '',
+    displayDenominations: {},
+    fiatAmount: () => '',
+    onSelect: () => {},
+    isHeaderRow: false,
+    isLastOfDate: false,
+    isSent: false
+  }
+
+  onSelect = () => {
+    this.props.onSelect(this.props.fioRequest)
+  }
+
+  requestedTimeAndMemo = (time: Date, memo: string) => {
+    const maxLength = 34
+    const memoStr = memo && memo.length > maxLength ? memo.slice(0, maxLength) + '... ' : memo
```

Comment:
React native supports auto ellipses for text. 

<img width="830" alt="Screen Shot 2020-04-21 at 5 50 41 PM" src="https://user-images.githubusercontent.com/6079354/79928755-ad3ff780-83f8-11ea-9480-e54a31baaa47.png">

Reply:
fixed